root/static/scripts/edit/components/ArtistCreditEditor.js

```diff
@@ -145,6 +145,24 @@ function addEmptyCredit(stateCtx: CowContext<StateT>) {
   setAutoJoinPhrases(namesCtx);
 }
 
+function swapCredits(
+  stateCtx: CowContext<StateT>,
```

Comment:
It wasn't obvious to me if there's a better way to write this function using `mutate-cow`, but please let me know if you can come up with something cleaner. :-)

Reply:
Welcome to the Cowverse! I didn't spot any improvements to be had here.